server/registry/actions_specs_test.go

```diff
@@ -477,6 +480,121 @@ func TestGetApiSpecContents(t *testing.T) {
 	}
 }
 
+func gZippedBytes(input []byte) ([]byte, error) {
+	var buf bytes.Buffer
+	zw, _ := gzip.NewWriterLevel(&buf, gzip.BestCompression)
```

Comment:
This should never return an error, but I think it's worth checking and returning anyway.

Reply:
agreed, done